README.md

```diff
@@ -39,11 +44,50 @@ discovery in human whole-genome sequencing (WGS) and exome sequencing data.
 - Summary Metrics
 
 ### Software version requirements :
-- GATK 4.beta.5 or later for PairedSingleSampleWF. GATK4.0.1.0 or later for JointGenotypingWf.
-- Picard 2.x
-- Samtools (see gotc docker)
+- GATK 4.beta.5 for PairedSingleSampleWF. GATK4.0.11.0 for JointGenotypingWf.
+- Picard 2.16.0-SNAPSHOT
+- Samtools 1.3.1
 - Python 2.7
+- Cromwell version support 
+  - Successfully tested on v37
+  - Does not work on versions < v23 due to output syntax
+
+### Important Note :
+- VQSR wiring. The SNP and INDEL models are built in parallel, but then the corresponding 
+  recalibrations are applied in series. Because the INDEL model is generally ready 
+  first (because there are fewer indels than SNPs) we set INDEL recalibration to 
+  be applied first to the input VCF, while the SNP model is still being built. By 
+  the time the SNP model is available, the indel-recalibrated file is available to 
+  serve as input to apply the SNP recalibration. If we did it the other way around, 
+  we would have to wait until the SNP recal file was available despite the INDEL 
+  recal file being there already, then apply SNP recalibration, then apply INDEL 
+  recalibration. This would lead to a longer wall clock time for complete workflow 
+  execution. Wiring the INDEL recalibration to be applied first solves the problem.
+- The current version of the posted "Generic germline short variant joint genotyping" 
+  is derived from the Broad production version of the workflow, which was adapted for 
+  large WGS callsets of up to 20K samples.  We believe the results of this workflow run 
+  on a single WGS sample are equally accurate, but there may be some shortcomings when 
+  the workflow is modified and run on small cohorts.  Specifically, modifying the SNP 
+  ApplyRecalibration step for higher specificity may not be effective.  The user can verify 
+  if this is an issue by consulting the gathered SNP tranches file.  If the listed 
+  truthSensitivity in the rightmost column is not well matched to the targetTruthSensitivity 
+  in the leftmost column, then requesting that targetTruthSensitivity from ApplyVQSR will 
+  not use an accurate filtering threshold.  This workflow has not been tested on exomes.  
+  The dynamic scatter interval creating was optimized for genomes.  The scattered SNP 
+  VariantRecalibration may fail because of two few "bad" variants to build the negative model. 
+  Also, apologies that the logging for SNP recalibration is overly verbose.
```

Comment:
Is it really the logging or just the commandline?

Reply:
Not sure, this message was passed down to me. Judging by an executed job from the [featured workspace](https://portal.firecloud.org/#workspaces/help-gatk/Germline-SNPs-Indels-GATK4-hg38/monitor/b28e64e4-cd49-4517-94ec-4676d0b6ea81/7c790b6a-1e80-48be-8858-e830a982d56b), neither look abnormally long.